src/components/form.scss

```diff
@@ -1,36 +1,114 @@
-%standard-input {
+%form-control-global {
+  // Public API (allowed to be overridden)
+  --_op-form-control-height-small: var(--op-input-height-small);
+  --_op-form-control-height-medium: var(--op-input-height-medium);
+  --_op-form-control-height-large: var(--op-input-height-large);
+  --_op-form-control-font-small: var(--op-font-x-small);
+  --_op-form-control-font-medium: var(--op-font-small);
+  --_op-form-control-font-large: var(--op-font-small);
+
   // Private API (don't touch these)
-  --__op-input-height: var(--op-input-height-large);
+  --__op-form-control-height: var(--_op-form-control-height-large);
+  --__op-form-control-font-size: var(--_op-form-control-font-large);
+
+  min-width: var(--__op-form-control-height);
+  height: var(--__op-form-control-height);
+
+  // Size Modifiers
+  &.form-control--small {
+    --__op-form-control-height: var(--_op-form-control-height-small);
+    --__op-form-control-font-size: var(--_op-form-control-font-small);
+  }
+
+  &.form-control--medium {
+    --__op-form-control-height: var(--_op-form-control-height-medium);
+    --__op-form-control-font-size: var(--_op-form-control-font-medium);
+  }
+
+  &.form-control--large {
+    --__op-form-control-height: var(--_op-form-control-height-large);
+    --__op-form-control-font-size: var(--_op-form-control-font-large);
+  }
+}
+
+%form-control-input-global {
```

Comment:
@Jeremy-Walton This looks good but we may want to enhance our documentation story around custom styling, and how to properly work with these tokens within "global" definitions.

Reply:
I did add a section to the form documentation related to these.
<img width="1071" alt="Screenshot 2023-05-20 at 5 17 29 PM" src="https://github.com/RoleModel/optics/assets/5957102/ca629110-3fc2-48d4-ac3c-30259d7b6723">